src/StudyUtils.in.jsm

```diff
@@ -1,5 +1,33 @@
 "use strict";
 
+/*
+* TODO bdanforth: Put these final questions at the top of the PR
+* @QUESTION glind: The function `mergeQueryArgs` accepts a rest parameter as its
+* second argument, but what is ultimately passed in as this parameter is an object
+* literal from 'endingQueryArgs'. When I execute `mergeQueryArgs` in a REPL,
+* passing in an object literal as the rest parameter, I get an error:
+*    "TypeError: objectLiteral is not iterable".
+* How does this `mergeQueryArgs` not throw an error for you?
+*/
+
+/**
+* STUDYUTILS OVERVIEW
+* This module takes care of the following:
+*  - Validates telemetry packets via JSON schema before sending pings
```

Comment:
for these bucket types:   'shield-study', 'shield-study-addon', 'shield-study-error'.  see docs/telemetry.md